expr/func_constantline.go

```diff
@@ -0,0 +1,54 @@
+package expr
+
+import (
+	"fmt"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/schema"
+)
+
+type FuncConstantLine struct {
+	value float64
+	from  uint32
+	to    uint32
+}
+
+func NewConstantLine() GraphiteFunc {
+	return &FuncConstantLine{}
+}
+
+func (s *FuncConstantLine) Signature() ([]Arg, []Arg) {
+	return []Arg{
+			ArgFloat{key: "value", val: &s.value},
+		}, []Arg{
+			ArgSeriesList{},
+		}
+}
+
+func (s *FuncConstantLine) Context(context Context) Context {
+	s.from = context.from - 1
+	s.to = context.to - 1
+	return context
```

Comment:
noticed all the timestamps were off by 1. have this hack for now - need to track down why this is happening though.

Reply:
This is the likely reason you see the discrepancy : https://github.com/grafana/metrictank/blob/ed6257fb34ac4ef7d09bae0c534c694fb64ada96/api/graphite.go#L1358